tests/utils.py

```diff
@@ -3,6 +3,25 @@
 
 
 class TestUtilities(unittest.TestCase):
+
+    def check_dashboard(self, driver, app_url):
+        """
+        Check if admin dashboard is available
```

Comment:
No, this is not required, I suppose a simple response of `200 OK` ensures the service is there, you don't need to do everything with selenium, this thing can even be done by a simply request: https://2.python-requests.org/en/master/

Moreover, this is only done once before starting tests, why is this in Utils, does this code need to be rerun? I can't think why I would rerun this! :sweat_smile:

Reply:
That's exactly what I think, I've asked it in my gitter channel to use `requests` 😅